integration/truncindex_test.go

```diff
@@ -115,11 +114,15 @@ func TestTruncIndex(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, cn, cStats.Attributes.Id)
 
-	t.Logf("Update container memory limit after started")
-	err = runtimeService.UpdateContainerResources(cnTruncIndex, &runtimeapi.LinuxContainerResources{
-		MemoryLimitInBytes: 50 * 1024 * 1024,
-	})
-	assert.NoError(t, err)
+	if goruntime.GOOS != "windows" {
+		// TODO(claudiub): remove this when UpdateContainerResources works on running Windows Containers.
```

Comment:
Might be worth referencing the ticket (#5187) for ease of future readers.

Reply:
Done